gnd/src/main/java/com/google/android/gnd/persistence/local/LocalDataStore.java

```diff
@@ -80,13 +81,25 @@
    */
   Completable applyAndEnqueue(ObservationMutation mutation);
 
+  /** Applies the specified {@link ObservationMutation} to the local data store.. */
+  Completable apply(ObservationMutation mutation) throws LocalDataStoreException;
+
+  /** Appends the specified {@link ObservationMutation} to the local queue for remote sync. */
+  Completable enqueue(ObservationMutation mutation);
+
+  /** Deletes local observation entry. */
+  Completable deleteObservation(ObservationMutation mutation);
```

Comment:
Isn't this just another case of `apply(ObservationMutation)`? Adding a separate method would break this model, and it would mean there are many configurations of ObservationMutation that would be invalid and would need to be checked. Can we instead call the impl of this method via `apply()`?

Reply:
Done 7f6dbff